plugins/mcreator-localization/help/default/entity/entity_data.md

```diff
@@ -0,0 +1,2 @@
+Add specific variables to your custom entity. 
+These variables are synchronised between the client and the server, meaning one of them can be used to change the texture of your entity.
```

Comment:
Why are you specifically pointing out switching entity textures? You said it can be used for many more things.
```suggestion
These variables are synchronized between client side and server side, so, unlike NBT tags, they can be used
to change any properties of this entity (for example, its texture).
```